linux_docker_resources/Dockerfile

```diff
@@ -175,11 +175,10 @@ RUN useradd -u 1234 -m rosbuild
 RUN sudo -H -u rosbuild -- git config --global user.email "jenkins@ci.ros2.org"
 RUN sudo -H -u rosbuild -- git config --global user.name "Jenkins ROS 2"
 RUN echo 'rosbuild ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
-WORKDIR "@workdir"
 
 # Add an entry point which changes rosbuild's UID from 1234 to the UID of the invoking user.
 # This means that the generated files will have the same ownership as the host OS user.
-ADD entry_point.sh /entry_point.sh
+COPY entry_point.sh /entry_point.sh
```

Comment:
The way we're using `ADD` wasn't that opaque but it hurt me to put an ADD back where a COPY would do.

[Docker's documentation](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) recommends COPY over ADD unless you need ADD's specific features.

> Although ADD and COPY are functionally similar, generally speaking, COPY is preferred. That’s because it’s more transparent than ADD. COPY only supports the basic copying of local files into the container, while ADD has some features (like local-only tar extraction and remote URL support) that are not immediately obvious. Consequently, the best use for ADD is local tar file auto-extraction...